src/controller.coffee

```diff
@@ -3672,6 +3690,16 @@ define ['droplet-helper',
 
     @redrawHighlights()
 
+  Editor::markBlock = (block, style) ->
+    key = @nextMarkedBlockId++
+
+    @markedBlocks[key] = {
+      model: block
+      style: style
+    }
```

Comment:
What's the reasoning for making markedBlocks an object with incrementing keys instead of just using an array?

i.e. we could do
```
@markedBlocks.push({model: block, style: style});
return @markedBlocks.length - 1